components/Menu.vue

```diff
@@ -22,3 +22,46 @@
 </script>
 
 
+<style lang="scss" scoped>
+  .menu {
+    background-color: white;
+    border-radius: 0.5rem;
+    box-shadow: 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05);
+    color: black;
+    font-size: 14px;
+    padding-top: 1.5rem;
+    width: 180px;
+    &-item {
+      margin-top: 1rem;
+      margin-left: 1.5rem;
+      &_svg {
+        display: inline-block;
+        margin-right: 1rem;
+        height: 0.75rem;
+        vertical-align: middle;
+        width: 0.75rem;
```

Comment:
When you write style think about it's sorting. 
Usually, I sort styles in this order:
1. Position rules
2. Display rules
3. Size rules
4. Block style rules
5. Font style rules
6. Other

Here is a plugin for sorting css, we can't use it with nuxt (I'm not sure). Btw look at their presets for sorting.
https://github.com/csscomb/csscomb.js - plugin
https://github.com/csscomb/csscomb.js/tree/dev/config - configs